File: gaia-sdk-javascript/src/graphql/request/input/CreateTenantImpulse.ts


import VariableRegistry from "../../../api/VariableRegistry"
import {Uuid, ISO8601, Struct} from "../../GaiaClient";
import {Order} from "../enumeration/Order";
import {OrderByField} from "../enumeration/OrderByField";
import {EdgeOrderByField} from "../enumeration/EdgeOrderByField";
import {EdgeType} from "../enumeration/EdgeType";

/**
 * The specification to create a tenant
 */
export class CreateTenantImpulse {
public _typeName = "CreateTenantImpulse";
    private qualifier:String;

    constructor (qualifier:String) {
        this.qualifier = qualifier;
    }
}

File: gaia-sdk-python/gaia_sdk/graphql/request/input/CreateTenantImpulse.py


from typing import Callable, List
from gaia_sdk.api.VariableRegistry import VariableRegistry
from gaia_sdk.graphql.request.enumeration.Order import Order
from gaia_sdk.graphql.request.enumeration.OrderByField import OrderByField
from gaia_sdk.graphql.request.enumeration.EdgeOrderByField import EdgeOrderByField
from gaia_sdk.graphql.request.enumeration.EdgeType import EdgeType


class CreateTenantImpulse():
    """
    The specification to create a tenant
    """
    qualifier: str

    def __init__(self, qualifier: str):
        self.qualifier = qualifier

    def __eq__(self, other):
        if type(other) is type(self):
            return self.qualifier == other.qualifier
        return False

    def __repr__(self):
        return {'qualifier': self.qualifier}
